presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -0,0 +1,173 @@
+
+.. function:: st_within(geometry, geometry) -> boolean
+
+    Returns true if and only if left geometry is within right geometry.
+
+Operations
+----------
+
+.. function:: st_boundary(geometry) -> geometry
+
+    Returns binary representation of the boundary geometry of input geometry.
+
+.. function:: st_buffer(geometry, double) -> geometry
+
+    Returns binary representation of the geometry buffered by distance.
+
+.. function:: st_difference(geometry, geometry) -> geometry
+
+    Returns binary representation of the geometry difference of left geometry and right geometry.
+
+.. function:: st_envelope(geometry) -> geometry
+
+    Returns binary representation of envelope of the input geometry.
+
+.. function:: st_exterior_ring(geometry) -> geometry
+
+    Returns binary representation of the exterior ring of the polygon.
+
+.. function:: st_intersection(geometry, geometry) -> geometry
+
+    Returns binary representation of the geometry intersection of left geometry and right geometry.
+
+.. function:: st_symmetric_difference(geometry, geometry) -> geometry
+
+    Returns binary representation of the geometry symmetric difference of left geometry and right geometry.
+
+Accessors
+---------
+
+.. function:: st_area(geometry) -> double
+
+    Returns area of input polygon.
+
+.. function:: st_centroid(geometry) -> varchar
+
+    Returns point that is the center of the polygon's envelope.
+
+.. function:: st_coordinate_dimension(geometry) -> bigint
+
+    Returns count of coordinate components.
+
+.. function:: st_dimension(geometry) -> bigint
+
+    Returns spatial dimension of geometry.
+
+.. function:: st_distance(geometry, geometry) -> double
+
+    Returns distance between left geometry and right geometry.
+
+.. function:: st_is_closed(geometry) -> boolean
+
+    Returns true if and only if the line is closed.
+
+.. function:: st_is_empty(geometry) -> boolean
+
+    Returns true if and only if the geometry is empty.
+
+.. function:: st_is_ring(geometry) -> boolean
+
+    Returns true if and only if the line is closed and simple.
+
+.. function:: st_length(geometry) -> double
+
+    Returns the length of line.
+
+.. function:: st_max_x(geometry) -> double
+
+    Returns the maximum X coordinate of geometry.
+
+.. function:: st_max_y(geometry) -> double
+
+    Returns the maximum Y coordinate of geometry.
+
+.. function:: st_min_x(geometry) -> double
+
+    Returns the minimum X coordinate of geometry.
+
+.. function:: st_min_y(geometry) -> double
+
+    Returns the minimum Y coordinate of geometry.
+
+.. function:: st_start_point(geometry) -> point
+
+    Returns the first point of an line.
+
+.. function:: st_end_point(geometry) -> point
+
+    Returns the last point of an line.
+
+.. function:: st_x(point) -> double
+
+    Returns the X coordinate of point.
+
+.. function:: st_y(point) -> double
+
+    Returns the Y coordinate of point.
+
+.. function:: st_point_number(geometry) -> bigint
```

Comment:
I think this was renamed to `st_point_count`

Reply:
@electrum , FWIW, PostGIS has this function under ST_NumPoints name: http://postgis.org/docs/ST_NumPoints.html

Reply:
OpenGIS standard also mentions ST_NumPoints and ST_NumInteriorRings